build/cqs.tsv

```diff
@@ -150,6 +156,8 @@ org.apache.hbase:hbase-protocol	1.3.1	compile
 org.apache.hbase:hbase-server	1.3.1	compile
 org.apache.hive:hive-storage-api	2.6.0	compile
 org.apache.htrace:htrace-core	3.1.0-incubating	compile
+org.apache.httpcomponents:httpclient	4.5.3	compile
+org.apache.httpcomponents:httpcore	4.4.6	compile
```

Comment:
Hmm... it is very odd that these versions do not match.

Reply:
httpcore only goes up to 4.4.11, but I could try lowering the httpclient version and see if it still works

Reply:
from looking at the poms, they don't always match. For instance, httpclient 4.5.9 (the latest) depends on httpcore 4.4.11

Reply:
do we actually need httpclient? It's not mentioned in the reporter libs, unless they are transitive deps: https://github.com/azagniotov/codahale-aggregated-metrics-cloudwatch-reporter#third-party-dependencies. But I notice they are explicitly declared in our pom.

Reply:
I can try taking it out and seeing if it works

Reply:
It did not work without httpclient 4.5.3, but I pushed the other changes back up